README.rst

```diff
@@ -100,7 +100,7 @@ To make it easier to build KWIVER, especially
 on systems like Microsoft Windows that do not have package manager,
 Fletch_ was developed to gather, configure and build dependent packages
 for use with KWIVER.  Fletch is a CMake_ based "super-build" that
-takes care of most of the build details for you.
+takes care of most of the built details for you.
```

Comment:
typo was fixed in the wrong spot.  This one should be "build".  Now there are two typos :)